ui/dashboards/src/components/Panel/PanelHeader.tsx

```diff
@@ -31,10 +31,9 @@ export interface PanelHeaderProps extends Omit<CardHeaderProps, OmittedProps> {
     onDuplicatePanelClick: () => void;
     onDeletePanelClick: () => void;
   };
-  isHovered: boolean;
```

Comment:
Taking away this prop since this was the only reason we needed it. We don't export `PanelHeader`, so this should not be a breaking change.